docker/package/tezos_baking_wizard.py

```diff
@@ -468,7 +468,16 @@ def bootstrap_node(self):
         self.import_snapshot()
 
         self.systemctl_start_action("node")
-        proc_call("sleep 5")
+
+        status_code = None
+        while status_code != 200:
```

Comment:
I'm not sure this condition is correct: in case a node is available (so no `URLError` is thrown), but the response is not `200` wouldn't this be stuck in an endless loop?

Reply:
Sure, but I would've thought it's intended, since we need an 'OK' response to proceed using the node. Do you think we should accept any response code as long as there is one and it's not a `URLError`?

Reply:
IMO yes, we should accept any successful connection here, regardless of the returned code.

An alternative might be to throw an error when the status code is not `200`, but I think we should limit this loop to only wait for the node RPC to be available for connection.
I think that whatever comes next should decide what to do with the content of the response (including the status code).